base/linalg/symmetric.jl

```diff
@@ -278,22 +278,116 @@ eigfact{T1<:Real,T2}(A::RealHermSymComplexHerm{T1,T2}) = (T = eltype(A); S = pro
 
 eigfact!{T<:BlasReal,S<:StridedMatrix}(A::RealHermSymComplexHerm{T,S}, irange::UnitRange) = Eigen(LAPACK.syevr!('V', 'I', A.uplo, A.data, 0.0, 0.0, irange.start, irange.stop, -1.0)...)
 # Because of #6721 it is necessary to specify the parameters explicitly here.
+
+"""
+    eigfact(A::Union{SymTridiagonal, Hermitian, Symmetric}, irange::UnitRange) -> Eigen
+
+Computes the eigenvalue decomposition of `A`, returning an `Eigen` factorization object `F`
+which contains the eigenvalues in `F[:values]` and the eigenvectors in the columns of the
+matrix `F[:vectors]`. (The `k`th eigenvector can be obtained from the slice `F[:vectors][:, k]`.)
+
+The following functions are available for `Eigen` objects: [`inv`](@ref), [`det`](@ref), and [`isposdef`](@ref).
+
+The `UnitRange` `irange` specifies indices of the sorted eigenvalues to search for.
+
+!!! note
+    If `irange` is not `1:n`, where `n` is the dimension of `A`, then the returned factorization
+    will be a *truncated* factorization and in this case, the determinant may be 0 and other methods
+    (such as [`inv`](@ref)) called on the factorization may throw.
```

Comment:
Just state that it is truncated for now. Right now the results of `inv` and `det` are wrong when the factorization is truncated.